cli/popper/commands/cmd_info.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+import click
+
+from popper import utils as pu
+from popper.cli import log, pass_context
+
+
+@click.argument('action', required=True)
+@click.option(
+    '--full',
+    help='Show full content of the README.',
+    is_flag=True
+)
+@click.option(
+    '--update-cache',
+    help='Update the action metadata cache before looking for info.',
+    is_flag=True
+)
+@click.command('info', short_help='Show details about an action.')
+@pass_context
+def cli(ctx, action, full, update_cache):
+
+    try:
+        org, repo = action.split('/')
+    except ValueError:
+        log.fail('Bad action name. Required format {org}/{repo}')
```

Comment:
here we can follow the same syntax as in the `uses` keyword. This with the goal of being consistent with how actions are referenced in `.workflow` files. This also means that we can support things like: `org/repo/path/to/action`.

And later we could support actions in arbitrary git repos, but we can leave that for later

Reply:
@ivotron We can utilize the `scm.parse()` for this. Since it is not implemented properly now, should I implement this now or fix it with #623 in a separate pr?

Reply:
that'd be awesome

Reply:
Thanks ! I will send the PR for #623 soon along with this fix.